### PR TITLE
Predictive Back Cross Fragment Shared Element Transition Sample

### PR DESCRIPTION
**Pull request by ashnohe**
Add a Predictive Back cross fragment shared element transition sample.

Video: https://photos.app.goo.gl/Et8F17wSZgmk5aNK9

### CODE REVIEW

File: samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBSharedElementTransitionFragment.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 The Android Open Source Project
```
Comment (yaraki) [inline samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBSharedElementTransitionFragment.kt:2]: nit: 2024

File: samples/user-interface/predictiveback/src/main/res/layout/fragment_shared_element_transition.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            android:layout_height="400dp"
+            android:gravity="center_vertical"
+            android:padding="16dp"
+            android:text="@string/sharedElementTransitionText"
+            android:textAppearance="?attr/textAppearanceBodyMedium"
+            android:textSize="16sp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
```
Comment (yaraki) [inline samples/user-interface/predictiveback/src/main/res/layout/fragment_shared_element_transition.xml:57]: Remove the `app:layout_constraint...` params in this TextView.

File: samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBSharedElementTransitionFragment.kt (hunk 1 above)
Comment (ashnohe) [reply samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBSharedElementTransitionFragment.kt]: done

File: samples/user-interface/predictiveback/src/main/res/layout/fragment_shared_element_transition.xml (hunk 2 above)
Comment (ashnohe) [reply samples/user-interface/predictiveback/src/main/res/layout/fragment_shared_element_transition.xml]: done

**Review by yaraki — APPROVED** [review]

**Pull request merged**